From a625ae4feb700b31747c26fed40cfeba89898e99 Mon Sep 17 00:00:00 2001
From: Tom Herbers <mail@tomherbers.de>
Date: Thu, 12 Dec 2024 17:30:30 +0100
Subject: [PATCH 1/3] modules: update openwrt

aa1241e979 wifi-scripts: fix failing mesh setup with missing wpa_supplicant
28de3ffefa netifd: check if /sbin/wifi exists before calling it
6c664135f9 netifd: update to Git 768027c5a7640137857f412f554227abd6160cdd (2024-11-27)
620aa80064 mediatek: MERCUSYS MR90X v1: add OpenWrt U-Boot (UBI) layout
24bddf68bd uboot-mediatek: add MERCUSYS MR90X v1 support
d79a70542b arm-trusted-firmware-mediatek: add mt7986-spim-nand-ddr3 build
d48e499fc1 mediatek: filogic: Migrate wifi configuration device paths
b9df2d79b6 mediatek: filogic: prevent faulty mac address assignment
5afd3daeba bpftool: turn off libbfd feature in host build
1eba94e67b mac80211: realtek: add RTL8812au/RTL8821au support
8a4ce69eea mac80211: realtek: backport support for RTL8812AU/RTL8821AU
2b45e93113 linux-firmware: realtek: add RTL8812AU/RTL8821AU firmware
4bfe5ccd11 linux-firmware: update to 20241110
863d1a634e qualcommax: ipq807x: add Netgear DEVICE_VARS
---
 modules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules b/modules
index 0fa4d2ea44..13c611d7a6 100644
--- a/modules
+++ b/modules
@@ -2,7 +2,7 @@ GLUON_FEEDS='gluon packages routing'
 
 OPENWRT_REPO=https://github.com/openwrt/openwrt.git
 OPENWRT_BRANCH=openwrt-24.10
-OPENWRT_COMMIT=45f9f1551223b516cccbcca1e252e722bb3dbc08
+OPENWRT_COMMIT=aa1241e9794c2c03b74f14fef8530507ee4fc2f7
 
 PACKAGES_GLUON_REPO=https://github.com/freifunk-gluon/packages.git
 PACKAGES_GLUON_COMMIT=3d08b0fee8dc5d96d8bcdb985fad1d5564de4022

From 36d907fe1a72db9d69d8e92c8c57277d59f7a9d1 Mon Sep 17 00:00:00 2001
From: Tom Herbers <mail@tomherbers.de>
Date: Thu, 12 Dec 2024 17:30:44 +0100
Subject: [PATCH 2/3] modules: update packages

441a0c18c v2raya: Update to 2.2.6.3
c92bf4d89 xray-core: Update to 24.11.21
9d9dc663e geoipupdate: Update to 7.1.0
fcbf91c99 cloudflared: Update to 2024.11.1
f2f6489c9 microsocks: backport upstream fixes
4a08b45f8 ksmbd-tools: update to version 3.5.3
039634159 python-dateutil: "downgrade" to version 2.9.0
---
 modules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules b/modules
index 13c611d7a6..b4ed2e1dc4 100644
--- a/modules
+++ b/modules
@@ -9,7 +9,7 @@ PACKAGES_GLUON_COMMIT=3d08b0fee8dc5d96d8bcdb985fad1d5564de4022
 
 PACKAGES_PACKAGES_REPO=https://github.com/openwrt/packages.git
 PACKAGES_PACKAGES_BRANCH=openwrt-24.10
-PACKAGES_PACKAGES_COMMIT=d364dcb7e55ac4202853c48e2788f0d4fbb5c513
+PACKAGES_PACKAGES_COMMIT=441a0c18cb6f9fb3295eb3d08cdc281c7dc7dc38
 
 PACKAGES_ROUTING_REPO=https://github.com/openwrt/routing.git
 PACKAGES_ROUTING_BRANCH=openwrt-24.10

From 879e4501be17aff41199fc71f73ff6e6d168506e Mon Sep 17 00:00:00 2001
From: Tom Herbers <mail@tomherbers.de>
Date: Thu, 12 Dec 2024 17:30:48 +0100
Subject: [PATCH 3/3] modules: update routing

eaa4aba batman-adv: Merge bugfixes from 2024.4
---
 modules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules b/modules
index b4ed2e1dc4..16b07abb22 100644
--- a/modules
+++ b/modules
@@ -13,4 +13,4 @@ PACKAGES_PACKAGES_COMMIT=441a0c18cb6f9fb3295eb3d08cdc281c7dc7dc38
 
 PACKAGES_ROUTING_REPO=https://github.com/openwrt/routing.git
 PACKAGES_ROUTING_BRANCH=openwrt-24.10
-PACKAGES_ROUTING_COMMIT=3f15699240c076d5ee9ed697fa5ef45355423f6f
+PACKAGES_ROUTING_COMMIT=d4125fbe02ad9996cc0367ca48f34f5cbf4a964f
